src/zipcrypto.rs

```diff
@@ -0,0 +1,158 @@
+use std::num::Wrapping;
+
+struct ZipCryptoKeys {
+    key_0: Wrapping<u32>,
+    key_1: Wrapping<u32>,
+    key_2: Wrapping<u32>,
+}
+
+impl ZipCryptoKeys {
+    //Used this paper to implement ZipCrypto algo
```

Comment:
Can this be part of the module documentation?

Reply:
Improved & fixed docstrings throughout the entire `zipcrypto.rs` document. Added module level documentation.